src/main/options.js

```diff
@@ -1,139 +1,81 @@
 "use strict";
 
 const path = require("path");
-
-const validate = require("jest-validate").validate;
-const deprecatedConfig = require("./deprecated");
-const getSupportInfo = require("../common/support").getSupportInfo;
+const supportInfo = require("../common/support").getSupportInfo(null, {
+  showDeprecated: true,
+  showUnreleased: true
+});
+const normalizer = require("./options-normalizer");
 const loadPlugins = require("../common/load-plugins");
 const resolveParser = require("./parser").resolveParser;
 const getPrinter = require("./get-printer");
 
-const defaults = {
-  cursorOffset: -1,
-  rangeStart: 0,
-  rangeEnd: Infinity,
-  useTabs: false,
-  tabWidth: 2,
-  printWidth: 80,
-  singleQuote: false,
-  trailingComma: "none",
-  bracketSpacing: true,
-  jsxBracketSameLine: false,
-  parser: "babylon",
-  parentParser: "",
-  insertPragma: false,
-  requirePragma: false,
-  semi: true,
-  proseWrap: "preserve",
-  arrowParens: "avoid",
-  plugins: [],
+const hiddenDefaults = {
   astFormat: "estree",
-  printer: {},
-  __inJsTemplate: false
+  printer: {}
```

Comment:
This was just to satisfy jest-validate. Might not be required now.

Reply:
It's also used to bypass the option validation, see L62.